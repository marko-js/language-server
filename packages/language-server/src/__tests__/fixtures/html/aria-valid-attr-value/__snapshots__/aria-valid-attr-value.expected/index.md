## Diagnostics
### Ln 1, Col 49
```marko
> 1 | <input type="checkbox" aria-label="my checkbox" aria-checked="sometimes">
    |                                                 ^^^^^^^^^^^^ Type '"sometimes"' is not assignable to type 'AttrTriState'.
  2 |
```

