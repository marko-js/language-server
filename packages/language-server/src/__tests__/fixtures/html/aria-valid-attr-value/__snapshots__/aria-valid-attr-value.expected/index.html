<input data-marko-node-id="0" type="checkbox" aria-label="my checkbox" aria-checked="sometimes">